includes/Core/Assets/Assets.php

```diff
@@ -212,10 +226,7 @@ public function enqueue_fonts() {
 
 		$this->fonts_enqueued = true;
 
-		$font_families = array(
-			'Google+Sans:300,300i,400,400i,500,500i,700,700i',
-			'Roboto:300,300i,400,400i,500,500i,700,700i',
-		);
```

Comment:
I don't think this line needed to change, we just want to add the filter afterwards to allow it to be filterable. Then after the filter is applied, if it's empty, we essentially abort the enqueue as mentioned in the ACs.